packages/client/hmi-client/src/workflow/ops/decapodes/tera-decapodes-node.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+	<div>
+		<Button label="Open" @click="emit('open-drilldown')" severity="secondary" outlined />
+	</div>
+</template>
+
+<script setup lang="ts">
+import { WorkflowNode } from '@/types/workflow';
+import Button from 'primevue/button';
+import { DecapodesOperationState } from './decapodes-operation';
+
+defineProps<{
+	node: WorkflowNode<DecapodesOperationState>;
+}>();
+
+const emit = defineEmits(['update-state', 'append-output', 'open-drilldown']);
+</script>
```

Comment:
Seem like very little thing is actual in-use: e.g. emit, props, maybe some clean up?